examples/valid/demo-orderapi-v3.oas2.yaml

```diff
@@ -7,9 +7,6 @@ info:
     email: "z@goodapi.co"
   description: |
     Sample API conforming to [adidas API Guidelines](https://adidas.gitbook.io/api-guidelines/).
-    
-    This API is [documented at Apiary](http://docs.demotemplate.apiary.io)
```

Comment:
We don't migrate the demo template to swaggerhub ?
By the way i don't see template feature in swaggerhub, can we have the same that we had in apairy ?

Reply:
not right now. However, I believe that was on the product roadmap for swaggerhub. We have a call with their product team next week or so to check the status on it.